packages/@ourworldindata/grapher/src/timeline/TimelineComponent.tsx

```diff
@@ -174,23 +191,73 @@ export class TimelineComponent extends React.Component<TimelineComponentProps> {
         }
     }
 
+    @computed private get areBothHandlesVisible(): boolean {
+        return this.controller.startTime !== this.controller.endTime
+    }
+
+    @computed private get shouldShowHoverTimeHandle(): boolean {
+        return (
+            !this.manager.isSingleTimeSelectionActive &&
+            !this.isDragging &&
+            !this.areBothHandlesVisible
+        )
+    }
+
+    private setHoverTime(event: any): void {
+        if (!this.shouldShowHoverTimeHandle) return
+
+        const inputTime = this.getInputTimeFromMouse(event)
+        if (!inputTime) return
+
+        if (!this.slider) return
+        const mouseX = getRelativeMouse(this.slider, event).x
+        const startX =
+            this.controller.startTimeProgress * this.slider.clientWidth
+        const handleX1 = startX - HANDLE_DIAMETER / 2
+        const handleX2 = startX + HANDLE_DIAMETER / 2
+
+        // Hide the hover handle when the mouse is positioned directly over
+        // the existing time handle
+        if (mouseX >= handleX1 && mouseX <= handleX2) {
+            this.hoverTime = undefined
+            return
+        }
```

Comment:
I was slightly confused about this code, and didn't immediately realise that `handleX1` and `handleX2` are the left and right bounds of the existing handle.

Maybe it makes sense to do a "clearer" (at least to me) bounds checking:
```ts
if (Math.abs(mouseX - startX) >= HANDLE_DIAMETER / 2)
```

Also, it might make sense to actually use the full `HANDLE_DIAMETER` instead, because the two handles will currently overlap by half:

<img width="398" height="172" alt="Image" src="https://github.com/user-attachments/assets/e651cade-df53-4279-ab3d-47ef15887cbf" />